avaje-config/src/main/java/io/avaje/config/Parsers.java

```diff
@@ -1,72 +0,0 @@
-package io.avaje.config;
```

Comment:
I suspect you removed this because you wanted to use Map in the public API. I don't think that is a good design option because Map is too easy to break [too fragile] in terms of future design changes.

Reply:
I removed it because I noticed that it was nothing but a glorified map.